data/videos.yml

```diff
@@ -3693,3 +3693,30 @@ Seattle Xcoders February 2016:
     title: Continuous Integrations for iOS Apps
     tags: [xcode, build, continuous integration]
     vimeo: 158290425
+NSLondon February 2016:
+  - language: English
+    speakers: [Raimon Lapuente, Gergely Orosz]
+    title: "Sorry Objc: It’s not you, it’s me"
```

Comment:
I can recreate the failure locally with `bundle exec middleman build --clean`. I didn’t find this when I initially made the PR because I just used `bundle exec middleman` — and in that case I don’t think you see the problem unless you try to view a single video page, and I think the only way to do this is via search.